pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/ESFile.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.dsm.model.elastic;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class ESFile {
```

Comment:
@pegahtah I have created such file as well when I did filtering refactoring which is still in PR, https://github.com/broadinstitute/ddp-study-server/blob/c763ecd6088954d1dde14f02135d99cbe1799e39/pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/elastic/Files.java could you please take a look at it and change this file accordingly to avoid conflicts in the future?

Reply:
I matched the naming of my changes to your file